content/docs/replication/deployment/powerflex.md

```diff
@@ -0,0 +1,255 @@
+> but it is recommended to install drivers on all clusters you will use for
+> replication.
+
+### Creating Storage Classes
+
+To be able to provision replicated volumes you need to create properly
+configured storage classes on both source and target clusters.
+
+Pair of storage classes on the source and target clusters would be essentially
+`mirrored` copies of one another. You can create them manually or with help from
+`repctl`.
+
+#### Manual Storage Class Creation
+
+You can find a sample of a replication enabled storage class in the driver repository
+[here](https://github.com/dell/csi-powerflex/blob/main/samples/storageclass/vxflexos-replication.yaml).
+
+It will look like this:
+```yaml
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: vxflexos-replication
+provisioner: csi-vxflexos.dellemc.com
+reclaimPolicy: Delete
+allowVolumeExpansion: true
+volumeBindingMode: Immediate
+parameters:
+  replication.storage.dell.com/isReplicationEnabled: "true"
+  replication.storage.dell.com/remoteStorageClassName: "vxflexos-replication"
+  replication.storage.dell.com/remoteClusterID: <remoteClusterID>
+  replication.storage.dell.com/remoteSystem: <remoteSystemID>
+  replication.storage.dell.com/remoteStoragePool: <remoteStoragePool>
+  replication.storage.dell.com/rpo: 60
+  replication.storage.dell.com/volumeGroupPrefix: "csi"
+  replication.storage.dell.com/consistencyGroupName: <desiredConsistencyGroupName>
+  replication.storage.dell.com/protectionDomain: <remoteProtectionDomain>
+```
+
+Let's go through each parameter and what it means:
+* `replication.storage.dell.com/isReplicationEnabled` if set to `true` will mark
+  this storage class as replication enabled, just leave it as `true`.
+* `replication.storage.dell.com/remoteStorageClassName` points to the name of
+  the remote storage class. If you are using replication with the multi-cluster
+  configuration you can make it the same as the current storage class name.
+* `replication.storage.dell.com/remoteClusterID` represents ID of a remote
+  cluster. It is the same id you put in the replication controller config map.
+* `replication.storage.dell.com/remoteSystem` is the name of the remote system
+  as seen from the current PowerFlex instance. This parameter is the systemID of
+  the array.
+* `replication.storage.dell.com/rpo` is an acceptable amount of data, which is
+  measured in units of time, that may be lost due to a failure.
+* `replication.storage.dell.com/volumeGroupPrefix` represents what string would
+  be appended to the volume group name to differentiate them.
+* `replication.storage.dell.com/consistencyGroupName` represents the desired
+  name to give the consistency group on the PowerFlex array. If omitted, the
+  driver will generate a name for the consistency group.
+* `replication.storage.dell.com/protectionDomain` represents the remote array's
+  protection domain to use.
+
+Let's follow up that with an example, let's assume we have two Kubernetes
```

Comment:
Replace comma with a full stop (period "."). Initial uppercase on "Let's" to start a new sentence.

Reply:
@falfaroc : If possible, could you please correct the grammar mistakes in other driver files as well?